RoaringBitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java

```diff
@@ -692,6 +692,53 @@ private int highestHigh() {
     return RoaringIntPacking.highestHigh(signedLongs);
   }
 
+  /**
+   * to be used with naivelazyor
+   */
+  public void repairAfterLazy() {
```

Comment:
I would move after `.naivelazyor`, to be consistent with RoaringBitmap methods ordering.